profiling/src/php_ffi.h

```diff
@@ -18,6 +18,8 @@
 // Profiling needs ZAI config for INI support.
 #include <config/config.h>
 
+#include <exceptions/exceptions.h>
```

Comment:
Can you leave a note about why this header is here like the other lines around it?